charts/stable/plex/SCALE/questions.yaml

```diff
@@ -108,7 +109,7 @@ questions:
           description: "Networks allowed to connect to plex"
           schema:
             type: string
-            default: ""
+            default: "172.16.0.0/16,YOURLOCALSUBNETIP"
```

Comment:
YOURLOCALSUBNETIP might cause issues by default, are you sure this is alright?
Also, maybe also add this to values.yaml?

Reply:
Not really sure how plex parses it, but i'll remove it for good measure, and change to `"172.16.0.0/16,192.168.1.0/24"` ?
And add on description `"Change 192.168.1.0/24'` to your local network ip?

Reply:
How about:
"Add your local network IP to this list" in the description.

It's not required to enter plex once connected to your account using a claimtoken.

Reply:
and just leave out the 192.168.1.0